contributors.md

```diff
@@ -27,3 +27,4 @@ Thank you to all who have provided guidance on the development of this software.
 - Michael Wetter, Lawrence Berkeley National Laboratory
 - Tao Yang, University of Southern Denmark
 - Ettore Zanetti, Lawrence Berkeley National Laboratory
+- Xing Lu, Pacific Northwest National Laboratory
```

Comment:
Please list your name in alphabetical order by last name.

Reply:
Changed